src/DependencyInjection/SonataUserExtension.php

```diff
@@ -320,4 +317,9 @@ private function checkManagerTypeToModelTypeMapping(array $config): void
             }
         }
     }
+
+    private function configureMailer(array $config, ContainerBuilder $container): void
+    {
+        $container->setAlias('sonata.user.mailer', $config['mailer']);
```

Comment:
Bonuspoint: Make an alias for the `MailerInterface` too

Reply:
We are implementing the FOS MailerInterface, this would lead to expose to autowiring the FOS MailerInterface from this bundle. Is this really what we want?